app/controllers/rsvp_controller.rb

```diff
@@ -104,6 +104,8 @@ def rsvp
           flash[:error] = "Sorry, but the capacity for that event has been reached."
         elsif @event_attendance.errors[:audience] && @event_attendance.errors[:audience].any?
           flash[:error] = "Invalid Audience, check that you are using the correct RSVP link."
+        elsif @event_attendance.errors[:base] && @event_attendance.errors[:base].any?
+          flash[:error] = @event_attendance.errors[:base].join(' ')
```

Comment:
I would recommend using `to_sentence` instead of `join(' ')` because it handles multiple error messages much better.